src/dat/gui/GUI.js

```diff
@@ -468,7 +468,7 @@ GUI.TEXT_CLOSED = 'Close Controls';
 GUI.TEXT_OPEN = 'Open Controls';
 
 GUI._keydownHandler = function(e) {
-  if (document.activeElement.type !== 'text' &&
+  if (document.activeElement && document.activeElement.type !== 'text' &&
```

Comment:
I think this is probably not the right behavior. If there is no active element, the keypress should toggle the GUI as it normally would.